### DISCUSSION TITLE
Unable to Authenticate with Github.com (Github Desktop)

### DISCUSSION THREAD

**Discussion by PotatoSnail** [community/community#84864] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

   In Github Desktop, I am able to sign in with Github, and commit to main on a repository, but when I try to push something, it says "Unable to authenticate with Github,com", and that I need to sign in with my username and password in the app. 

   This would not be an issue, however, when I try to sign in, no matter what, it always tells me "authentication failed". I have even reset my password in case that was the problem, but it still does not work. What can I do to fix this?
![image](https://github.com/community/community/assets/98859196/9fcb3fa8-7742-41f3-ba88-a7585522eb7a)
This has never happened before, and I have had no problems like this in the past.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @PotatoSnail ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
